src/views/tickets/tpp-return-to-cart.php

```diff
@@ -1,4 +1,18 @@
 <?php
+/**
+ * This template renders the "Return to Cart" text.
+ *
+ * Override this template in your own theme by creating a file at:
+ *
+ *     [your-theme]/tribe-events/tickets/tpp-return-to-cart.php
+ *
+ * @package TribeEventsCalendar
```

Comment:
@sc0ttkclark what's the name of the package we want to use?

Reply:
@juanfra we haven't been setting package on these template files, we generally haven't included that and could probably remove the line. Maybe a good question for TEC dev channel to see what standard of usage we want to go with there for template views.

Reply:
Thanks Scott! @rafsuntaskin I guess removing that line would be the way to go then!